r2i/tflite/engine.cc

```diff
@@ -230,5 +226,100 @@ Engine::~Engine () {
   this->Stop();
 }
 
+template <typename T>
+static uint8_t ConvertToFixedPoint(const T value, const TfLiteTensor &tensor) {
```

Comment:
Interface segregation principle: don't pass in unnecessary info. Just pass in  `scale` and `zero`.

Reply:
Also, this is a nice generic function, I'll make it even more generic by returning a templated type as well.